driver-core/src/test/java/com/datastax/driver/core/CCMBridge.java

```diff
@@ -1075,16 +1105,16 @@ private String randomizePorts(CharSequence str) {
         }
 
         @Override
-        @SuppressWarnings("SimplifiableIfStatement")
         public boolean equals(Object o) {
-            // do not include cluster name and start, only
```

Comment:
I think we should keep the comment about cluster name because indeed a cluster name is unique, but two CCM instances should still be considered equal if their characteristics are identical. As for the `start` property, I am not sure anymore, but I think we can include it the computation of equals() and hashcode().